src/resolution.rs

```diff
@@ -257,12 +289,14 @@ mod test {
         );
 
         assert_eq!(
-            resolve("Folder", "SourceAssetId", "532413"),
+            resolve("IntValue", "Value", "532413"),
             Variant::Int64(532413),
         );
 
         assert_eq!(resolve("Part", "Transparency", "1"), Variant::Float32(1.0));
         assert_eq!(resolve("NumberValue", "Value", "1"), Variant::Float64(1.0));
+
+        assert_eq!(resolve_unambiguous("1.337"), Variant::Float64(1.337));
```

Comment:
```suggestion
        assert_eq!(resolve_unambiguous("12.5"), Variant::Float64(12.5));
```

When checking for exact floating point equality, we should use values that can definitely be resolved exactly to prevent futures issues related to rounding.